src/client/wetty/term.ts

```diff
@@ -20,6 +21,14 @@ export class Term extends Terminal {
     this.fitAddon = new FitAddon();
     this.loadAddon(this.fitAddon);
     this.loadAddon(new WebLinksAddon());
+    /**
+     * FIXMEs:
+     * - worker file prolly needs some better delivering story than just
+     *   being hardcopied to assets folder
+     * - apply some useful addon settings suitable for wetty
+     * - maybe introduce windows switch here (no ConPTY support for SIXEL atm)
+     */
+    this.loadAddon(new ImageAddon('/wetty/assets/xterm-addon-image-worker.js'));
```

Comment:
I wonder if the file could be pulled In from web_modules but this definite works given the add on is still in testing

Reply:
Well didnt quite figure out how web_modules gets composedor pulls things from packages, thats why the assets hack. :smile: 

How could it be done directly via web_modules?

Reply:
Btw the sixel path is pretty consolidated (in beta), the alpha/experimental is still on the module due other pending image support (at least png and jpg is planned).
For wetty the worker approach is def. a nuisance (due to having only one terminal in a document), there is an issue open to also provide a non workered version (no clue yet how to shape that neatly).

Reply:
For reference the web_modules is built by snowpack it parses out the node_modules imported by the clients moves them to the web_modules folder and updates the import path.
The server then just serves them and I leave it up to the browsers to handle the imports and caching of js files.

Reply:
Thx for the explanation. I messed around with snowpack for a while and tried to get working in a more straight forward way:
- packing the worker file in separate package `xterm-addon-image-worker` does not work, as snowpack decorates the worker with an import declaration, which is not yet supported by all browser for workers, maybe there is a special worker flag to avoid that (did not find any)
- generate a `worker.ts` in wetty project and import image worker there hoping it will handle it all right - nope, same import error as above

Finally this works:
- code as `this.loadAddon(new ImageAddon('/wetty/web_modules/pkg/xterm-addon-image-worker.js'));`
- yarn build
- `cp node_modules/xterm-addon-image/lib/xterm-addon-image-worker.js.map build/web_modules/pkg/`
- yarn start

Again this manual copy is annoying, maybe this could be integrated into snowpack build flow easier. Not sure if thats any better than treating the worker file as a static js asset.

Reply:
Snowpack can just do moves and regular commands. But I think for this case the vendor ing of the file its fine to get the feature rolled out.

If we need to update the file in the future we can re look at how to manually add the dependency. If all else fails moving back to bash will always work.

Reply:
So moving it "hard" to assets (and exclude it from linter) will do for now?

Reply:
Yep